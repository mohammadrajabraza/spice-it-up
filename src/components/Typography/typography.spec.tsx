import setupComponentSnapshotTests from 'storybook/utils/setup-component-snapshot-tests';

import typographyConfig from './typography.config';

setupComponentSnapshotTests(typographyConfig);
