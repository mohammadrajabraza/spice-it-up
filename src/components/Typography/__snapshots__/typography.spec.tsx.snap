// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given - Typography component and props are: {"children":"This is a text","variant":"body1"} should render correctly and match snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#1C1B1B",
        "fontSize": 32,
        "fontWeight": "normal",
        "letterSpacing": 0.5,
        "lineHeight": 44,
      },
      {},
    ]
  }
>
  This is a text
</Text>
`;

exports[`Given - Typography component and props are: {"children":"This is a text","variant":"body2"} should render correctly and match snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#1C1B1B",
        "fontSize": 26,
        "fontWeight": "normal",
        "letterSpacing": 0.5,
        "lineHeight": 44,
      },
      {},
    ]
  }
>
  This is a text
</Text>
`;

exports[`Given - Typography component and props are: {"children":"This is a text","variant":"body3"} should render correctly and match snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#1C1B1B",
        "fontSize": 23,
        "fontWeight": "normal",
        "letterSpacing": 0.3,
        "lineHeight": 36,
      },
      {},
    ]
  }
>
  This is a text
</Text>
`;

exports[`Given - Typography component and props are: {"children":"This is a text","variant":"button"} should render correctly and match snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#1C1B1B",
        "fontSize": 26,
        "fontWeight": "normal",
        "lineHeight": 32,
      },
      {},
    ]
  }
>
  This is a text
</Text>
`;

exports[`Given - Typography component and props are: {"children":"This is a text","variant":"caption"} should render correctly and match snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#3B3B3B",
        "fontSize": 19,
        "fontWeight": "normal",
        "letterSpacing": 0.5,
      },
      {},
    ]
  }
>
  This is a text
</Text>
`;

exports[`Given - Typography component and props are: {"children":"This is a text","variant":"heading1"} should render correctly and match snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#1C1B1B",
        "fontSize": 43,
        "fontWeight": "bold",
        "letterSpacing": 0.5,
        "lineHeight": 55,
        "verticalAlign": "middle",
      },
      {},
    ]
  }
>
  This is a text
</Text>
`;

exports[`Given - Typography component and props are: {"children":"This is a text","variant":"heading2"} should render correctly and match snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#1C1B1B",
        "fontSize": 32,
        "fontWeight": "bold",
        "letterSpacing": 0.5,
        "lineHeight": 41,
        "verticalAlign": "middle",
      },
      {},
    ]
  }
>
  This is a text
</Text>
`;

exports[`Given - Typography component and props are: {"children":"This is a text","variant":"heading3"} should render correctly and match snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#1C1B1B",
        "fontSize": 23,
        "fontWeight": "bold",
        "letterSpacing": 0.5,
        "lineHeight": 32,
        "verticalAlign": "middle",
      },
      {},
    ]
  }
>
  This is a text
</Text>
`;
